Gemfile

```diff
@@ -18,13 +18,13 @@
 source "https://rubygems.org"
 ruby RUBY_VERSION
 
-gem "jekyll", "4.2.0"
+gem "jekyll", "4.3.3"
```

Comment:
I tested updating jekyl locally and found it let me build the site -- see https://github.com/apache/arrow-site/pull/451 for a PR with just that change